templates/components/itilobject/timeline/form_document_item.html.twig

```diff
@@ -171,7 +165,6 @@
                         'multiple': true,
                         'full_width': true,
                         'is_horizontal': false,
-                        'add_field_html': max_size,
```

Comment:
This should not be removed.

Reply:
This is not useful. The max upload size is already displayed within the file upload box.
![Selection_434](https://github.com/user-attachments/assets/b4983671-6769-4679-83e8-f2aa7543956d)